### PR TITLE
10.30

### PR DESCRIPTION
**Pull request by ZiningFan00**
how to plot likert data

### CODE REVIEW

**Comment by bojum** [comment]
[edited]
Hi Zining and Xinyuan. Your pull request is missing some important files. We'll have to ask you to do the following. 

- Fetch from jtr13/cc19 and merge into your local branch. See [here](https://edav.info/github.html#nd-nth-pr-on-another-repo-with-branching) for more info.
- Commit the change in your local branch.
- Push your local branch to your origin.

There will be a lot more files now, but don't worry. That's just what we want. Let us know if you have any questions.

**Comment by ZiningFan00** [comment]
I created a new pull request #156 from my master. Does the new one work now?

**Comment by ZiningFan00** [comment]
I changed my file and commited again, I double checked my process and it seems right. Does it work now? Thank you.

**Comment by bojum** [comment]
To be specific, we are looking for DESCRIPTION and _bookdown.yml which should be available automatically when you fork/fetch from jtr13/cc19. I still do not see those files in this pull request....

**Comment by skyetim** [comment]
Did you delete all the original files after you forked the repo? If so, please fork again, don't delete any files, and follow the instructions on the website to add your contribution to it.

**Comment by ZiningFan00** [comment]
Oh, I see the problem now. I just forked the repo from jtr13/cc19 and there are no other files. I didn't delete the files. I don't know why it would like this. Let me try again.

**Comment by ZiningFan00** [comment]
Can I delete the forked repo on my account?

**Comment by skyetim** [comment]
revised in #159
